business/istio_validations.go

```diff
@@ -157,42 +153,40 @@ func (in *IstioValidationsService) GetIstioObjectValidations(ctx context.Context
 	errChan := make(chan error, 1)
 
 	// Get all the Istio objects from a Namespace and all gateways from every namespace
-	wg.Add(9)
+	wg.Add(7)
 	go in.fetchNamespaces(ctx, &namespaces, errChan, &wg)
-	go in.fetchIstioConfigList(ctx, &istioConfigList, namespace, errChan, &wg)
-	go in.fetchExportedResources(ctx, &exportedResources, namespace, errChan, &wg)
+	go in.fetchIstioConfigList(ctx, &istioConfigList, &mtlsDetails, &rbacDetails, namespace, errChan, &wg)
 	go in.fetchServices(ctx, &services, namespace, errChan, &wg)
 	go in.fetchWorkloads(ctx, &workloads, namespace, errChan, &wg)
```

Comment:
I have a similar thought here with the "fetchWorkloads" and "fetchAllWorkloads".
Do we need to keep both?

I know it's technical debt, but I think this work is a good opportunity to improve it.

Reply:
Good point, better to merge here.